docs/install_guides/installing-openvino-apt.md

```diff
@@ -131,19 +132,43 @@ sudo apt install openvino-opencv-<VERSION>.<UPDATE>.<PATCH>
 
 After you have installed OpenVINO Runtime, if you decided to [install OpenVINO Development Tools](installing-model-dev-tools.md), make sure that you install external software dependencies first. 
 
-Refer to <a href="installing-openvino-linux.md#install-external-dependencies">Install External Software Dependencies</a> for detailed steps.
+Refer to <a href="openvino_docs_install_guides_installing_openvino_linux.html#install-external-dependencies">Install External Software Dependencies</a> for detailed steps.
+
+
+### Step 5 (Optional): Configure Inference on Non-CPU Devices
+
+@sphinxdirective
+
+.. tab:: GPU
+
+   To enable the toolkit components to use processor graphics (GPU) on your system, follow the steps in :ref:`GPU Setup Guide <gpu guide>`.
+
+.. tab:: NCS 2
+
+   To perform inference on Intel® Neural Compute Stick 2 powered by the Intel® Movidius™ Myriad™ X VPU, follow the steps on :ref:`NCS2 Setup Guide <ncs guide>`.
+   <!--For more details, see the `Get Started page for Intel® Neural Compute Stick 2 <https://software.intel.com/en-us/neural-compute-stick/get-started>`.-->
+
+.. tab:: VPU
+
+   To install and configure your Intel® Vision Accelerator Design with Intel® Movidius™ VPUs, see the :ref:`VPU Configuration Guide <vpu guide>`.
```

Comment:
for some reason I don't see a link to `VPU Configuration Guide`, it's just a text, while for NCS2 and GPU we have links. The same for all other documents.

Reply:
Changes made in another file got lost when copying all the changes to this branch. :) Fixed.